pkg/utils/utils.go

```diff
@@ -346,3 +361,36 @@ func Chroot(path string) (func() error, error) {
 		return syscall.Chroot(".")
 	}, nil
 }
+
+func RunCommand(command string, args ...string) (string, error) {
+	glog.V(2).Infof("RunCommand(): %s %v", command, args)
+	var stdout, stderr bytes.Buffer
+
+	cmd := exec.Command(command, args...)
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+
+	err := cmd.Run()
+	glog.V(2).Infof("RunCommand(): %s", stdout.String())
+	return stdout.String(), err
+}
+
+func GetNicSriovMode(pciAddress string) (string, error) {
+	glog.V(2).Infof("GetNicSriovMode(): device %s", pciAddress)
+	args := []string{"dev", "eswitch", "show", fmt.Sprintf("pci/%s", pciAddress)}
+	out, err := RunCommand("devlink", args...)
```

Comment:
Why not using the go netlink lib DevLinkSetEswitchMode and DevLinkGetDeviceByName see https://github.com/vishvananda/netlink/blob/fd97bf4e47867b5e794234baa6b8a7746135ec10/devlink_linux.go